sql/expression/function/load_file_test.go

```diff
@@ -0,0 +1,130 @@
+package function
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/dolthub/go-mysql-server/sql"
+	"github.com/dolthub/go-mysql-server/sql/expression"
+)
+
+func createTempDirAndFile(fileName string) (string, *os.File, error) {
+	dir := os.TempDir()
+
+	file, err := ioutil.TempFile(dir, fileName)
+	if err != nil {
+		return "", nil, err
+	}
+
+	return dir, file, nil
+}
+
+func TestLoadFileNoSecurePriv(t *testing.T) {
+	// Create a valid temp file and temp directory
+	_, file, err := createTempDirAndFile("myfile.txt")
+	assert.NoError(t, err)
+
+	_, err = file.Write([]byte("my data"))
+	assert.NoError(t, err)
+
+	fileName := expression.NewLiteral(file.Name(), sql.Text)
+	fn := NewLoadFile(sql.NewEmptyContext(), fileName)
+
+	// Assert that Load File returns the regardless since secure_file_priv is set to an empty directory
+	res, err := fn.Eval(sql.NewEmptyContext(), sql.Row{})
+	assert.NoError(t, err)
+	assert.Equal(t, []byte("my data"), res)
+
+	err = file.Close()
+	assert.NoError(t, err)
+
+	err = os.Remove(file.Name())
```

Comment:
Both the `Close` and `Remove` should be deferred right after creating the file. If ever there were a panic then we'd never remove the file, creating garbage. It's good practice to always defer a file's closure immediately after opening, even when the proceeding code should always work. You never know how it may be modified in the future.

Reply:
I did but for every test case except for the for loop one